Gemfile

```diff
@@ -5,7 +5,7 @@ gemspec
 
 group :test do
   gem 'actionpack'
-  gem 'logstash-event', '1.1.5'
+  gem 'logstash-event', git: 'https://github.com/elasticsearch/logstash.git', tag: '1.5'
```

Comment:
I'd forgotten about that particular idiosyncracy of logstash. :sweat: 

Yeah, it might be worth a small mention in the gemspec, just as a pointer for folks who aren't aware. It might save a few moments head-scratching.